### DISCUSSION TITLE
v4.5.5

### DISCUSSION THREAD

**Discussion by anhoder** [go-musicfox/go-musicfox#324] [General]
### Features
* 05701f0 feat: remove go-optional dep
* bdfd213 feat(ui)!: refactor player song management
* c64db0d feat: better random play
* 9e8e957 feat: add maxPlayErrCount config
### Bug fixes
* d287b9e fix: 心动模式与一般模式切换
* 7decce4 fix: crash in random mode when playlist is empty
* e2edb32 fix: like song heart display
* 0a320eb fix: incomplete merge
* 691caae fix: crash in macOS <=10.15 #255
* 1071b91 fix: 修改注释
* c7bb51b fix: migu concurrent write header #299
* 0025af1 fix: index out of range error
### Optimize
* 8a0a546 optimize: cache使用os.UserCacheDir
### Docs
* 7f56dab docs: update README.md
### CI
* f52a19f fix build
* d5d3eaa fix darwin build
* a9b3f8f fix windows build
### Others
* 3578e94 chore: update scoop config & changelog
* 6bd5740 chore: clean
* 91403ab chore: update default cfg
* 6a85a0e 一键下载歌词功能键位更改
* 552fc0c 一键下载歌词功能
* 5589a71 ci: remove `close-issue-no-response`
* 1b074e7 ci: update close-issue-no-response.yaml
* f244c46 ci: update cron
* f3d2e7e ci: fix repo
* 1b6d2c6 ci: create close-issue-no-response.yaml
* 012eeae Merge pull request #298 from Yefori-Go/master
* 3f72ee4 Merge pull request #295 from yigekuyou/XDG规范
* 4b07f42 chore: update scoop config & changelog

<hr /><em>This discussion was created from the release <a href='https://github.com/go-musicfox/go-musicfox/releases/tag/v4.5.5'>v4.5.5</a>.</em>

**Comment by crossterra-agent** [comment]
- **Status**: IN_PROGRESS
- **Role**: orchestrator
- **Step**: 
- **Link**: https://github.com/CrossTerra/monorepo/pull/2021
- **Summary**: Validator APPROVED — created PR with 5-step plan: https://github.com/CrossTerra/monorepo/pull/2021
